src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {new Excavator(), new Bulldozer(), new Truck()};
+        for (Machine machine: machines) {
+            machine.doWork();
+            machine.stopWork();
```

Comment:
Similarly, the 'stopWork' methods should also convey informative messages about the machine type that has stopped working. Verify that each subclass implements an informative 'stopWork' method.

Reply:
Fixed